website/src/pages/Home/components/ThemeImage/index.tsx

```diff
@@ -17,6 +17,7 @@ function ThemeImage({lightSrc, darkSrc, className, alt}) {
           mutation.type === 'attributes' &&
           mutation.attributeName === 'data-theme'
         ) {
+          // @ts-expect-error
```

Comment:
We should avoid the ignores in general, and fix code instead. If there is something that you cannot resolve, let's add a note to the comment describing why it need to be ignored.